documentation/asciidoc/microcontrollers/rp2040/rp2040_based_boards.adoc

```diff
@@ -0,0 +1,60 @@
+== RP2040-based Boards
+
+=== Boards from Raspberry Pi
+
+image::images/pico.jpg[]
+
+https://www.raspberrypi.org/products/raspberry-pi-pico/[Raspberry Pi Pico]:: Designed by Raspberry Pi, RP2040 features a dual-core Arm Cortex-M0+ processor with 264KB internal RAM and support for up to 16MB of off-chip Flash. A wide range of flexible I/O options includes I2C, SPI, and — uniquely — Programmable I/O (PIO). These support endless possible applications for this small and affordable package.
+
+=== Boards from Adafruit
+
+image::images/feather-2040.jpg[]
+
+https://www.adafruit.com/product/4884[Feather 2040]:: A new chip means a new Feather, and the Raspberry Pi RP2040 is no exception. When we saw this chip we thought "this chip is going to be awesome when we give it the Feather Treatment" and so we did! This Feather features the RP2040, and all niceties you know and love about Feather.
+
+image::images/itsybitsy-2040.jpg[]
+
+https://www.adafruit.com/product/4888[ItsyBitsy 2040]:: A new chip means a new ItsyBitsy, and the Raspberry Pi RP2040 is no exception. When we saw this chip we thought "this chip is going to be awesome when we give it the ItsyBitsy teensy-weensy Treatment" and so we did! This Itsy' features the RP2040, and all niceties you know and love about the ItsyBitsy family.
+
+=== Boards from Arduino
+
+image::images/arduino_nano.png[]
+
+Nano RP2040 Connect:: Meet the only connected RP2040 board, leveraging essential key features of the other Arduino Nanos — like the versatile u-blox NINA WiFi and Bluetooth module. The new board comes packed with some high-quality MEMS sensors from STM, an efficient power section, and a bunch of other innovations.
+
+=== Boards from Pimoroni
+
+image::images/pga2040.png[]
```

Comment:
This needs to be `images/pga2040.jpg`

Reply:
Fixed in 5781295. But they're not **all** wrong, and none of the images are copied to the HTML directory?

Reply:
Changing `.png` to `.jpg` didn't help, which it certainly should have done! Also, thinking about it. Shouldn't that have thrown an error when it *tried* to copy the file to the HTML directory and couldn't find it? There wasn't an error generated.

Reply:
> Shouldn't that have thrown an error when it tried to copy the file to the HTML directory and couldn't find it?

It **did** throw an error about the missing file when I was fixing #2070 and that's how I spotted that this filename was wrong :wink: